DigitalLearningSolutions.Web/Views/TrackingSystem/CourseSetup/ManageCourse/EditLearningPathwayDefaults.cshtml

```diff
@@ -0,0 +1,86 @@
+﻿@inject IConfiguration configuration
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.CourseSetup.CourseDetails
+@using Microsoft.Extensions.Configuration
+@model EditLearningPathwayDefaultsViewModel
+
+@{
+  var errorHasOccurred = !ViewData.ModelState.IsValid;
+  ViewData["Title"] = "Edit Learning Pathway defaults";
+  ViewData["Application"] = "Tracking System";
+  ViewData["HeaderPath"] = $"{configuration["AppRootPath"]}/TrackingSystem/Centre/Dashboard";
+  ViewData["HeaderPathName"] = "Tracking System";
+  var cancelLinkData = new Dictionary<string, string> { { "customisationId", Model.CustomisationId.ToString() } };
+}
+
+@section NavMenuItems {
+  <partial name="/Views/TrackingSystem/Shared/_NavMenuItems.cshtml" />
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    @if (errorHasOccurred) {
+      <vc:error-summary order-of-property-names="@new []{ nameof(EditLearningPathwayDefaultsViewModel.CompleteWithinMonths), nameof(EditLearningPathwayDefaultsViewModel.ValidityMonths) }" />
+    }
+
+    <h1 class="nhsuk-heading-xl">Edit Learning Pathway defaults</h1>
+
+    <form class="nhsuk-u-margin-bottom-3" method="post"
+          asp-controller="ManageCourse"
+          asp-action="SaveLearningPathwayDefaults"
+          asp-route-customisationId="@Model.CustomisationId">
+      <div class="nhsuk-form-group">
+        <vc:text-input asp-for="@nameof(Model.CompleteWithinMonths)"
+                       label="Completion within (in months)"
+                       populate-with-current-value="true"
+                       type="text"
+                       spell-check="false"
+                       autocomplete=""
+                       hint-text="Within how many months of enrolment should the learner complete the course?"
+                       css-class="nhsuk-input nhsuk-input--width-10" />
+      </div>
+
+      <div class="nhsuk-form-group">
+        <vc:text-input asp-for="@nameof(Model.ValidityMonths)"
+                       label="Completion valid for (in months)"
+                       populate-with-current-value="true"
+                       type="text"
+                       spell-check="false"
+                       autocomplete=""
+                       hint-text="For how many months after completion will the completion remain valid
+                                                      (before the learner is expected to complete the course again)?"
+                       css-class="nhsuk-input nhsuk-input--width-10" />
+      </div>
+
+      <div class="nhsuk-checkboxes nhsuk-u-margin-bottom-3">

```

Comment:
I think that component has a non nullable label for the group of checkboxes so it's meant more for checkboxes that are specifically related to each other, whereas I feel like these are independent.

I've added more space between them, how does this look?
![localhost_5001_TrackingSystem_CourseSetup_24286_Manage_LearningPathwayDefaults (3)](https://user-images.githubusercontent.com/70278044/135586331-c2078e32-6254-4faf-9a72-3f0cc3193d6d.png)

Reply:
I think the spacing looks good here.

I see what you mean about the checkboxes view component as well. For reference, I don't believe that view components can have nullable/optional parameters, but you can put an empty string in (though in this case it would still create some HTML without any content). David is implementing a single-checkbox view component in 512, so if that ends up getting merged before this, it might be worth including it, but for now its probably fine as is.

Reply:
Oh right, of course you could just leave the label blank :) I will use David's checkbox component once it's merged!